pkg/job/job.go

```diff
@@ -17,10 +18,16 @@ import (
 	"github.com/insomniacslk/xjson"
 )
 
+// CurrentDescriptorVersion is the current version of the job descriptor
+// that the client must speaks to descripe jobs.
+// It has to number to denote breaking and non-breaking changes
+const CurrentDescriptorVersion string = "1.0"
```

Comment:
nit; this would make it easier to use in the code and avoids the (non-failing) conversions

```suggestion
const JobDescriptorMajorVersion uint = 1
const JobDescriptorMinorVersion uint = 0
```